app/views/propertyrepresentation/revokeAgentProperties.scala.html

```diff
@@ -55,7 +55,7 @@
         <input class="govuk-checkboxes__input" id="@id" name="linkIds[]" type="checkbox" value="@value">
         <label class="govuk-label govuk-checkboxes__label" for="@id">
             <span class="govuk-visually-hidden">Appoint</span>
-            @Formatters.capitalisedAddress(address)
+            <span data-address="address-text">@Formatters.capitalisedAddress(address)</span>
```

Comment:
it doesn't need to have a value. `<span data-address>...</span>` can be used to label the address spans

Reply:
I see - I hadn't come across these data- attributes before. I'll update now